### ISSUE TITLE
[Bug]: Scheduled tasks fail instantly

### ISSUE THREAD

**Issue by kaitallaoua** [intro-skipper/intro-skipper#332]
### Self service debugging

- [X] I use Jellyfin 10.9.11 (or newer) and my permissions are correct

### Describe the bug

I checked the three items in the troubleshooting guide, chromaprint seems good. Jellyfin otherwise streams/transcodes fine, ffmpeg looks good. I did the `sudo dpkg-statoverride --update --add jellyfin root 0644 /usr/share/jellyfin/web/index.html`, rebooted. Still same issue. `jellyfin-ffmpeg6` is installed. See the jellyfin log, not sure what the problem is: `Could not find a part of the path '/var/lib/jellyfin/data/introskipper/chromaprints/470e11668101d29154117a45756fef77-blackframes-1453.5-1454-v1`

### Jellyfin install method

https://tteck.github.io/Proxmox/ jellyfin LXC

### Container image/tag or Jellyfin version

10.9.11

### Operating System

Ubuntu 22.04 LXC

### Support Bundle

```shell
* Jellyfin version: 10.9.11
* Plugin version: 1.0.0+2d3a7fd3f711
* Queue contents: 27797 episodes, 1694 seasons
* Warnings: `UnableToAddSkipButton`
* FFmpeg: `okay`

FFmpeg version:

ffmpeg version 6.0.1-Jellyfin Copyright (c) 2000-2023 the FFmpeg developers
built with gcc 11 (Ubuntu 11.4.0-1ubuntu1~22.04)
configuration: --prefix=/usr/lib/jellyfin-ffmpeg --target-os=linux --extra-version=Jellyfin --disable-doc --disable-ffplay --disable-ptx-compression --disable-static --disable-libxcb --disable-sdl2 --disable-xlib --enable-lto --enable-gpl --enable-version3 --enable-shared --enable-gmp --enable-gnutls --enable-chromaprint --enable-opencl --enable-libdrm --enable-libxml2 --enable-libass --enable-libfreetype --enable-libfribidi --enable-libfontconfig --enable-libbluray --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvorbis --enable-libopenmpt --enable-libdav1d --enable-libsvtav1 --enable-libwebp --enable-libvpx --enable-libx264 --enable-libx265 --enable-libzvbi --enable-libzimg --enable-libfdk-aac --arch=amd64 --enable-libshaderc --enable-libplacebo --enable-vulkan --enable-vaapi --enable-amf --enable-libvpl --enable-ffnvcodec --enable-cuda --enable-cuda-llvm --enable-cuvid --enable-nvdec --enable-nvenc
libavutil      58.  2.100 / 58.  2.100
libavcodec     60.  3.100 / 60.  3.100
libavformat    60.  3.100 / 60.  3.100
libavdevice    60.  1.100 / 60.  1.100
libavfilter     9.  3.100 /  9.  3.100
libswscale      7.  1.100 /  7.  1.100
libswresample   4. 10.100 /  4. 10.100
libpostproc    57.  1.100 / 57.  1.100
```
```


### Jellyfin logs

```shell
[2024-10-06 23:07:23.922 -06:00] [INF] Scheduled Task is starting
[2024-10-06 23:07:24.006 -06:00] [INF] Running enqueue of items in library "Anime"
[2024-10-06 23:07:24.383 -06:00] [INF] Running enqueue of items in library "TV Shows"
[2024-10-06 23:07:25.855 -06:00] [INF] Running enqueue of items in library "Movies"
[2024-10-06 23:07:25.864 -06:00] [INF] [Mode: Credits] Analyzing 1 files from "91 Days" season 0
[2024-10-06 23:07:25.866 -06:00] [ERR] SaveTimestamps "Could not find a part of the path '/var/lib/jellyfin/data/introskipper/credits.xml'."
[2024-10-06 23:07:25.868 -06:00] [INF] [Mode: Credits] Analyzing 12 files from "91 Days" season 1
[2024-10-06 23:07:25.871 -06:00] [ERR] SaveTimestamps "Could not find a part of the path '/var/lib/jellyfin/data/introskipper/credits.xml'."
[2024-10-06 23:07:27.709 -06:00] [ERR] Error executing Scheduled Task
System.AggregateException: One or more errors occurred. (Could not find a part of the path '/var/lib/jellyfin/data/introskipper/chromaprints/470e11668101d29154117a45756fef77-blackframes-1453.5-1454-v1'.) (Could not find a part of the path '/var/lib/jellyfin/data/introskipper/chromaprints/eb04a89ab7603bfa390d05fb27316af8-credits'.)
 ---> System.IO.DirectoryNotFoundException: Could not find a part of the path '/var/lib/jellyfin/data/introskipper/chromaprints/470e11668101d29154117a45756fef77-blackframes-1453.5-1454-v1'.
   at Interop.ThrowExceptionForIoErrno(ErrorInfo errorInfo, String path, Boolean isDirError)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, UnixFileMode openPermissions, Int64& fileLength, UnixFileMode& filePermissions, Boolean failForSymlink, Boolean& wasSymlink, Func`4 createOpenException)
   at System.IO.File.OpenHandle(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)
   at System.IO.File.WriteAllBytes(String path, Byte[] bytes)
   at ConfusedPolarBear.Plugin.IntroSkipper.FFmpegWrapper.GetOutput(String args, String cacheFilename, Boolean stderr, Int32 timeout)
   at ConfusedPolarBear.Plugin.IntroSkipper.FFmpegWrapper.DetectBlackFrames(QueuedEpisode episode, TimeRange range, Int32 minimum)
   at ConfusedPolarBear.Plugin.IntroSkipper.Analyzers.BlackFrameAnalyzer.AnalyzeMediaFiles(IReadOnlyList`1 analysisQueue, AnalysisMode mode, CancellationToken cancellationToken)
   at ConfusedPolarBear.Plugin.IntroSkipper.ScheduledTasks.BaseItemAnalyzerTask.AnalyzeItems(IReadOnlyList`1 items, AnalysisMode mode, CancellationToken cancellationToken)
   at ConfusedPolarBear.Plugin.IntroSkipper.ScheduledTasks.BaseItemAnalyzerTask.<>c__DisplayClass5_0.<AnalyzeItems>b__4(KeyValuePair`2 season)
   at System.Threading.Tasks.Parallel.<>c__DisplayClass43_0`2.<PartitionerForEachWorker>b__1(IEnumerator& partitionState, Int64 timeout, Boolean& replicationDelegateYieldedBeforeCompletion)
--- End of stack trace from previous location ---
   at System.Threading.Tasks.Parallel.<>c__DisplayClass43_0`2.<PartitionerForEachWorker>b__1(IEnumerator& partitionState, Int64 timeout, Boolean& replicationDelegateYieldedBeforeCompletion)
   at System.Threading.Tasks.TaskReplicator.Replica.Execute()
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.TaskReplicator.Run[TState](ReplicatableUserAction`1 action, ParallelOptions options, Boolean stopOnFirstFailure)
   at System.Threading.Tasks.Parallel.PartitionerForEachWorker[TSource,TLocal](Partitioner`1 source, ParallelOptions parallelOptions, Action`1 simpleBody, Action`2 bodyWithState, Action`3 bodyWithStateAndIndex, Func`4 bodyWithStateAndLocal, Func`5 bodyWithEverything, Func`1 localInit, Action`1 localFinally)
--- End of stack trace from previous location ---
   at System.Threading.Tasks.Parallel.PartitionerForEachWorker[TSource,TLocal](Partitioner`1 source, ParallelOptions parallelOptions, Action`1 simpleBody, Action`2 bodyWithState, Action`3 bodyWithStateAndIndex, Func`4 bodyWithStateAndLocal, Func`5 bodyWithEverything, Func`1 localInit, Action`1 localFinally)
   at ConfusedPolarBear.Plugin.IntroSkipper.ScheduledTasks.BaseItemAnalyzerTask.AnalyzeItems(IProgress`1 progress, CancellationToken cancellationToken, IReadOnlyCollection`1 seasonsToAnalyze)
   at ConfusedPolarBear.Plugin.IntroSkipper.ScheduledTasks.DetectCreditsTask.ExecuteAsync(IProgress`1 progress, CancellationToken cancellationToken)
   at Emby.Server.Implementations.ScheduledTasks.ScheduledTaskWorker.ExecuteInternal(TaskOptions options)
 ---> (Inner Exception #1) System.IO.DirectoryNotFoundException: Could not find a part of the path '/var/lib/jellyfin/data/introskipper/chromaprints/eb04a89ab7603bfa390d05fb27316af8-credits'.
   at Interop.ThrowExceptionForIoErrno(ErrorInfo errorInfo, String path, Boolean isDirError)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, UnixFileMode openPermissions, Int64& fileLength, UnixFileMode& filePermissions, Boolean failForSymlink, Boolean& wasSymlink, Func`4 createOpenException)
   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
   at ConfusedPolarBear.Plugin.IntroSkipper.FFmpegWrapper.CacheFingerprint(QueuedEpisode episode, AnalysisMode mode, List`1 fingerprint)
   at ConfusedPolarBear.Plugin.IntroSkipper.FFmpegWrapper.Fingerprint(QueuedEpisode episode, AnalysisMode mode, Int32 start, Int32 end)
   at ConfusedPolarBear.Plugin.IntroSkipper.FFmpegWrapper.Fingerprint(QueuedEpisode episode, AnalysisMode mode)
   at ConfusedPolarBear.Plugin.IntroSkipper.Analyzers.ChromaprintAnalyzer.AnalyzeMediaFiles(IReadOnlyList`1 analysisQueue, AnalysisMode mode, CancellationToken cancellationToken)
   at ConfusedPolarBear.Plugin.IntroSkipper.ScheduledTasks.BaseItemAnalyzerTask.AnalyzeItems(IReadOnlyList`1 items, AnalysisMode mode, CancellationToken cancellationToken)
   at ConfusedPolarBear.Plugin.IntroSkipper.ScheduledTasks.BaseItemAnalyzerTask.<>c__DisplayClass5_0.<AnalyzeItems>b__4(KeyValuePair`2 season)
   at System.Threading.Tasks.Parallel.<>c__DisplayClass43_0`2.<PartitionerForEachWorker>b__1(IEnumerator& partitionState, Int64 timeout, Boolean& replicationDelegateYieldedBeforeCompletion)
--- End of stack trace from previous location ---
   at System.Threading.Tasks.Parallel.<>c__DisplayClass43_0`2.<PartitionerForEachWorker>b__1(IEnumerator& partitionState, Int64 timeout, Boolean& replicationDelegateYieldedBeforeCompletion)
   at System.Threading.Tasks.TaskReplicator.Replica.Execute()<---

[2024-10-06 23:07:27.710 -06:00] [INF] "Detect Credits" Failed after 0 minute(s) and 3 seconds
```

**Comment by kaitallaoua** [comment]
The `/var/lib/jellyfin/data/introskipper/chromaprints/` does not exist. There should be no permission issues with my media. In the jellyfin lxc itself, I never had to modify anything and I dont see any specifc instructions if there was something to do.
```
# ls -alh /var/lib/jellyfin/data
total 187M
drwxr-xr-x  6 jellyfin jellyfin   14 Oct  6 22:56 .
drwxr-x---  8 jellyfin adm         8 Oct  1 22:15 ..
drwxr-xr-x 12 jellyfin jellyfin   12 Oct  2 00:13 attachments
-rw-r--r--  1 jellyfin jellyfin   35 Oct  1 22:15 device.txt
-rw-r--r--  1 jellyfin jellyfin 176K Oct  6 22:55 jellyfin.db
-rw-r--r--  1 jellyfin jellyfin  32K Oct  6 23:14 jellyfin.db-shm
-rw-r--r--  1 jellyfin jellyfin 137K Oct  6 23:14 jellyfin.db-wal
-rw-r--r--  1 jellyfin jellyfin 342M Oct  6 23:02 library.db
-rw-r--r--  1 jellyfin jellyfin 1.4M Oct  6 23:02 library.db-shm
-rw-r--r--  1 jellyfin jellyfin 128M Oct  6 23:02 library.db-wal
drwxr-xr-x  2 jellyfin jellyfin    2 Oct  1 22:15 playlists
drwxr-xr-x  2 jellyfin jellyfin   19 Oct  6 21:57 ScheduledTasks
-rw-r--r--  1 jellyfin jellyfin 3.6M Oct  6 23:02 splashscreen.png
drwxr-xr-x  4 jellyfin jellyfin    4 Oct  2 00:12 subtitles
```

**Comment by rlauuzo** [comment]
This issue should be resolved in version 1.0.0.5. Please update to that version.

**Comment by kaitallaoua** [comment]
yes it works now thanks.
